jaeger-core/README.md

```diff
@@ -76,6 +74,17 @@ environment approach described above.
 
 More information about using the `TracerResolver` can be found [here](../jaeger-tracerresolver/README.md).
 
+#### Reporting internal metrics via Micrometer
+
+The Jaeger Java Client collects internal metrics and is able to report them via [Micrometer](http://micrometer.io).
+To accomplish that, include the artifact `com.uber.jaeger:jaeger-micrometer` as a dependency to your project and use
+`MicrometerStatsReporter` like this:
+
+```java
+MicrometerStatsReporter metricsReporter = new MicrometerStatsReporter();
```

Comment:
This example needs to be updated to use the `MicrometerStatsFactory` approach.

Reply:
Fixed